gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/ConditionMapperTest.java

```diff
@@ -188,6 +189,30 @@ public void testLinkSetWithNoDatesIsMappedCorrectly() {
         assertThat(conditions.get(0).getAssertedDateElement().getValue()).isNull();
     }
 
+    @Test
+    public void testLinkSetWithEffectiveTimeLowNullFlavorUnkIsMappedCorrectly() {
+        when(dateTimeMapper.mapDateTime(any())).thenReturn(EHR_EXTRACT_AVAILABILITY_DATETIME);
+        final RCMRMT030101UK04EhrExtract ehrExtract = unmarshallEhrExtract("linkset_with_null_flavor_unk.xml");
```

Comment:
2 super minor points.

The name of the fixture is a bit vague. A lot of things can have a null flavor UNK.

I also find "mappedCorrectly" is going to be true for any of our unit tests (I know you're following the pattern from above), "IsMappedWithNullOnsetDateTime" would be a lot more clear.